src/sentry/features/__init__.py

```diff
@@ -215,6 +215,9 @@
 # Workflow 2.0 Project features
 default_manager.add("projects:auto-associate-commits-to-release", ProjectFeature)
 
+# register the hard-coded handler
+default_manager.add_handler(ActiveReleaseDefaultOnHardcodeFeatureHandler())  # NOQA
```

Comment:
Question about this, the only way to enable and disable this would be to remove the handler? 

We no longer have the ability to toggle it off and on through flagr, right?

Reply:
Correct